src/app/scheduling/datetimeeditor/datetimeeditor-sample-1/datetimeeditor-sample-1.component.scss

```
@@ -0,0 +1,6 @@
+
+
+.sample-wrapper {
```

in the scss leave only padding that is consistent with the other datepicker/timepicker samples and rename the commonly used across the samples .sample-wrapper  class to more specific one like .datetime-wrapper

```suggestion
 .datetime-wrapper {
    padding: 1rem;
}

```